packages/headless/src/api/search/search/result.ts

```diff
@@ -172,4 +172,9 @@ export interface Result {
    * TopResult
    */
   rankingModifier?: string;
+
+  /**
+   * Whether the result item was previously viewed by the user specified in the request of the query.
```

Comment:
```suggestion
   * Whether the user who triggered the request has previously viewed the result item.
```

At least I _**believe**_ that's what this description is saying 🙂

Reply:
What exactly toggles this value? I believe there should be a link here to some docs.

Also, if someone views a result then immediately triggers another query with the same result, is this value already going to be `true`, or is there a delay? If there's a delay, I'm not sure that we should include this at all, or at-least we should include a very visible warning. Implementers will think that this value should be `true` in cases where it isn't and they will open support tickets.

Reply:
You can see the field being part of the Search API response schema here: 
![image](https://github.com/coveo/ui-kit/assets/2488155/4f8af846-5d4e-49bc-a21f-c860d77b9d73)

Reply:
I see. Then, I think the description of this property should be edited to either:
* Specify that this is affected by the `canSeeUserProfileOf` parameter specifically.
* A link to some docs that includes the above clarification.

I really want to avoid any potential confusion around this.

Reply:
description of the property updated 👌

Reply:
Do you think this is good enough @btaillon ?